source/JS/maps.js

```diff
@@ -88,4 +88,10 @@ function getPicCoords(callback)
             alert("Geocode was not successful for the following reason: " + status);
         }
     });
+}
+
+function addProfileBanner(){
+    //Create the search box and link it to the UI element.
```

Comment:
I think the idea was to mimic the way the search box was pushed onto the map and do something similar with the users name